module/migration.js

```diff
@@ -0,0 +1,30 @@
+/* global ui, game */
+
+let worldVersion;
+
+export const migrateWorld = async () => {

```

Comment:
Allow to migrate some data. We only use it once to migrate old characters to vampire type. 
But It's a useful tool to have, to change data.